src/BraketSimulator.jl

```diff
@@ -601,6 +602,104 @@ include("dm_simulator.jl")
         #pragma braket result expectation x(q[0]) 
         #pragma braket result variance x(q[1]) 
     """
+    
+    vqe_qasm = """
+        OPENQASM 3.0;
+        bit[2] b;
+        qubit[2] q;
+        ry(-4.97894242803364) q[0];
+        ry(-4.435983179322655) q[1];
+        cz q[0], q[1];
+        ry(6.249142469550989) q[0];
+        ry(2.509637558409141) q[1];
+        cz q[0], q[1];
+        ry(-5.476946260844031) q[0];
+        ry(5.937223228770655) q[1];
+        cz q[0], q[1];
+        ry(1.6839702128823246) q[0];
+        ry(-3.211915934619051) q[1];
+        b[0] = measure q[0];
+        b[1] = measure q[1];
+        """
+    all_gates_qasm = """
```

Comment:
Not much to do in this PR, but I wish there were a better way to generate this / avoid it becoming stale. Would be nice if the BDK exposed some enumeration of the gates.

Reply:
I think referencing the `supported_operations` part of `properties` might be a nice solution since that is available at package compile time.

Reply:
@rmshaffer Want to open an issue for that? So we don't lose track